### DISCUSSION TITLE
请问参考文献的样式该如何修改

### DISCUSSION THREAD

**Discussion by ginkgotyranny** [nju-lug/NJUThesis#71] [Q&A] [answered]
通过mathscinet下载bibtex后，将相应数据放在njuthesis-sample.bib文件中：
![@ CMI~}P)95(6Q~V%F78M94](https://user-images.githubusercontent.com/77227575/153559550-1d36d9ae-c0ef-44bd-94a9-11c3f72318b4.png)
编译后得到的参考文献样式为：
![image](https://user-images.githubusercontent.com/77227575/153559634-f38dd782-8b8b-4075-8e68-ae2a87897e29.png)
请问该如何修改，可以将参考文献样式修改为以下形式：
![image](https://user-images.githubusercontent.com/77227575/153559738-99ebf1c8-f673-4d2a-88be-174a1aeaa6b8.png)
主要修改：
1.作者的姓名缩写的顺序变化
2.作者姓名只有首字母为大写

**Comment by atxy-blip** [comment, marked as answer]
模板采用的是符合 GB/T 7714-2015 标准的参考文献表，姓名顺序、大写字母都出自有据可循的规定。目前能选的样式只包括该标准下的两种。就我个人理解而言，用于申请国内学位的论文直接套用国家标准即可。

改样式则需要找一个符合你要求的 `biblatex` 样式（而不是从头造轮子）。模板内相关支持还没写，一时半会还改不掉。如果你*确实*需要做这个修改的话请反馈给我。

-----

<https://github.com/nju-lug/NJUThesis/commit/134b829fd1a49805921ad58d95ed13b0d07e1a77> 已添加自由选择参考文献样式的功能，将随 v0.15.0 发布。
